lib/utils.js

```diff
@@ -53,3 +53,8 @@ export async function loadGetInitialProps (Component, ctx) {
   }
   return props
 }
+
+export function getLocationOrigin () {
+  const { protocol, hostname, port } = window.location
+  return `${protocol}//${hostname}${port ? ':' + port : ''}`
```

Comment:
Was thinking that too, but the escaped nested template string seemed a little funky. Or am I missing something? 
```javascript
return `${protocol}//${hostname}${port ? /`:${port}/` : ''}`
```